component-playground/config.js

```diff
@@ -11,7 +11,17 @@ var resolvePath = function(userPath) {
   return path.resolve(cwd, userPath);
 };
 
-module.exports = {
+var config;
+
+try {
```

Comment:
There seems no way, but I improved my code, because it would suppress any syntax errors from the custom config file.